internal/version/version.go

```diff
@@ -1,3 +1,31 @@
 package version
 
+import (
+	"runtime/debug"
+)
+
 const Version = "0.2.0"
+
+// GetVersion returns
+// the version number
+func GetVersion() string {
+	return Version
+}
+
+// GetShortRevision returns a short
+// commit or checkout revision
+// from build information
+func GetShortRevision() string {
+	if info, ok := debug.ReadBuildInfo(); ok {
```

Comment:
Go `1.18` allow for version control information. https://pkg.go.dev/runtime/debug#ReadBuildInfo